features/chat/components/chat-room-header.tsx

```
@@ -0,0 +1,267 @@
+export function ChatRoomHeader({ room }: ChatRoomHeaderProps) {
+  const [memberSearch, setMemberSearch] = useState('');
+  const participants = room.participants ?? [];
+  const visibleParticipants = participants.slice(0, MAX_VISIBLE_AVATARS);
+  const overflowCount = Math.max(0, participants.length - MAX_VISIBLE_AVATARS);
+
+  const RoomIcon =
+    room.type === ChatRoomType.ai
+      ? Bot
+      : room.type === ChatRoomType.group
+        ? Folder
+        : User;
```

ESLint

[error] Nested ternary expression should be parenthesized.

(unicorn/no-nested-ternary)